nodejs/pulumi/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "pulumi",
```

Comment:
@jkodroff to avoid any confusion with other modules under the `pulumi` namespace (https://www.npmjs.com/search?q=pulumi) could this module be named `pulumi-dagger-example` or similar?

Reply:
Fixed.